frontend/src/components/Sidebar/DistrictMap.tsx

```diff
@@ -32,7 +32,7 @@ interface IRegionPolygon {
 
 export default function DistrictMap(): JSX.Element {
   const [geodata, setGeodata] = useState<GeoJSON.GeoJSON | null>(null);
-  //const selectedDistrict = useAppSelector((state) => state.dataSelection.district);
+  const selectedDistrict = useAppSelector((state) => state.dataSelection.district);
```

Comment:
These changes have nothing to do with this PR.
```suggestion
  //const selectedDistrict = useAppSelector((state) => state.dataSelection.district);
```